Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -67,6 +67,15 @@ extension SPAppDelegate {
     func configureVersionsController() {
         versionsController = VersionsController(bucket: simperium.notesBucket)
     }
+
+    @objc
+    func configurePublishController() {
+        publishController = PublishController()
+        publishController.onUpdate = {
+            (note) in
+            PublishNoticePresenter.presentNotice(for: note)
+        }
```

Comment:
Nipticky: the capture statement is usually placed right by the `{` (rather than in the first closure line)

```
     publishController.onUpdate = { note in
             PublishNoticePresenter.presentNotice(for: note)
         }
```